enclave_manager/Dockerfile

```diff
@@ -278,7 +291,7 @@ WORKDIR /project/avalon/tc/sgx/trusted_worker_manager/common
 RUN mkdir -p build \
   && cd build \
   && cmake .. \
-  && make
+ && make
```

Comment:
You deleted this space. Restore the space to keep the same indentation as the previous lines.